#goal definition
float32 goal_x
float32 goal_y
---
#result definition

---
#feedback definition
float32 curr_x
float32 curr_y
float32 curr_theta
